plugins/module_utils/waiters.py

```diff
@@ -514,6 +563,18 @@ def rds_model(name):
         core_waiter.NormalizedOperationMethod(
             ec2.describe_subnets
         )),
+    ('EC2', 'vpc_available'): lambda ec2: core_waiter.Waiter(
+        'vpc_available',
+        ec2_model('VpcAvailable'),
+        core_waiter.NormalizedOperationMethod(
+            ec2.describe_vpcs
+        )),
+    ('EC2', 'vpc_exists'): lambda ec2: core_waiter.Waiter(
```

Comment:
There is already a waiter for managing this as well https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Waiter.VpcExists
Does it make sense to you?